cypress/integration/Crosscutting/BackToTopTest/index.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="Cypress" />
+import { When, Then, And } from "cypress-cucumber-preprocessor/steps";
+
+When('user scrolls 600 px down', () => {
+    cy.scrollTo(0, 600);
+});
+
+Then('back-to-top arrow appears with text {string} and href {string}', (textTop, hrefTop) => {
+    cy.get('div.site-footer__container > a').should('have.text', textTop).and('have.attr', 'href', hrefTop);
+});
+
+When('user scrolls up 100 px', () => {
+    cy.scrollTo(10, 0);
```

Comment:
the number should not be hardcoded - please pass it as parameter

Reply:
done